src/main/java/com/spotify/docker/client/DefaultDockerClient.java

```diff
@@ -1957,6 +1959,27 @@ public Task inspectTask(final String taskId) throws DockerException, Interrupted
   }
 
   @Override
+  public NodeInfo inspectNode(final String nodeId) throws DockerException, InterruptedException {
+    assertApiVersionIsAbove("1.24");
+
+    WebTarget resource = resource().path("nodes")
+        .path(nodeId);
+
+    try {
+      return request(GET, NodeInfo.class, resource, resource.request(APPLICATION_JSON_TYPE));
+    } catch (DockerRequestException e) {
+      switch (e.status()) {
+        case 404:
+          throw new NodeNotFoundException(nodeId);
+        case 503:
```

Comment:
[Docs for API 1.24](https://docs.docker.com/engine/api/v1.24/#inspect-a-node) say this is 406 while those for [1.28](https://docs.docker.com/engine/api/v1.29/#operation/NodeInspect) say 503. If 1.24 docs are correct, is it worth handling for that old API version?